### DISCUSSION TITLE
How to use "selection mode=' partial '"

### DISCUSSION THREAD

**Discussion by zdsystem** [bcakmakoglu/vue-flow#1641] [Q&A]
I have a requirement that requires a mouse to select nodes, but setting selection mode="partial" does not take effect

**Comment by bcakmakoglu** [comment]
The partial selection mode is for usage with the selection box (shift + mouse) so nodes that aren’t fully wrapped in a selection box are still included in it.

You might be looking for the multiselection key code which is the meta key by default
